docs/sdk/wallets/auth-methods.md

```diff
@@ -9,7 +9,9 @@ Authentication methods are ways of asigning Programmable Key Pairs (PKP) to a sp
 
 An authentication method refers to the specific credential (i.e a wallet address, Google oAuth, or Discord account) that is programmatically tied to the PKP and used to control the underlying key-pair. Only the auth method associated with a particular PKP has the ability to combine the underlying shares. You can read more about how authentication works with PKPs on our [blog](https://spark.litprotocol.com/how-authentication-works-with-pkps/).
 
-Right now, there are two main ways to do auth with Lit Actions. We will dive into these two methods below.
+Right now, there are two main ways to do auth with Lit Actions. We will dive into these two methods below:
+1. [Using Lit Auth Directly](#using-lit-auth-directly)
+2. [Custom Auth / Adding new Auth Methods not yet supported by Lit](#custom-auth--adding-new-auth-methods-not-yet-supported-by-lit)
```

Comment:
here let's just link to the custom auth page directly, not another redirect

Reply:
Done @a1ttech :)